tiiuae/ghaf-givc
# Add wifi switch service

## Description of changes

Turn On/Off functions are added.
Error outputs are fixed.

## Testing

- Replace ssid and password in the command and run grpcurl to connect to AP: `grpcurl '{"SSID":"ssid","Password":"password"}' -cacert /run/givc/ca-cert.pem -cert /etc/givc/gui-vm.ghaf-cert.pem -key /etc/givc/gui-vm.ghaf-key.pem 192.168.101.1:9000 systemd.WifiService.ConnectNetwork`.
- Observe the network connection on the Waybar. 
- Use grpcurl to turn off the wifi: `grpcurl -cacert /run/givc/ca-cert.pem -cert /etc/givc/gui-vm.ghaf-cert.pem -key /etc/givc/gui-vm.ghaf-key.pem 192.168.101.1:9000 systemd.WifiService.TurnOff`.
- Observe the disconnection on the Waybar. 
- Repeat the connection process, Check the connection and `No network with SSID found` error message.
- Use grpcurl to turn on the wifi back: `grpcurl -cacert /run/givc/ca-cert.pem -cert /etc/givc/gui-vm.ghaf-cert.pem -key /etc/givc/gui-vm.ghaf-key.pem 192.168.101.1:9000 systemd.WifiService.TurnOn`.
- Observe the successful network connection again.
